src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1

```diff
@@ -0,0 +1,154 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+<#
+.Synopsis
+Migrate logins from the source Sql Servers to the target Azure Sql Servers.
+.Description
+Migrate logins from the source Sql Servers to the target Azure Sql Servers.
+#>
+
+
+function New-AzDataMigrationLoginsMigration
+{
+    [OutputType([System.Boolean])]
+    [CmdletBinding(PositionalBinding=$false)]
+    [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Description('Migrate logins from the source Sql Servers to the target Azure Sql Servers.')]
+
+    param(
+        [Parameter(ParameterSetName='CommandLine', Mandatory, HelpMessage='Required. Connection string(s) for the source SQL instance(s), using the formal connection string format.')]
+        [System.String[]]
+        ${SourceSqlConnectionString},
+
+
+        [Parameter(ParameterSetName='CommandLine', Mandatory, HelpMessage='Required. Connection string(s) for the target SQL instance(s), using the formal connection string format.')]
+        [System.String]
+        ${TargetSqlConnectionString},
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Optional. Location of CSV file of logins. Use only one parameter between this and listOfLogins.')]
+        [System.String]
+        ${CSVFilePath},
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Optional. List of logins in string format. If large number of logins need to be migrated, use CSV file option.')]
+        [System.String[]]
+        ${ListOfLogins},
```

Comment:
Please use singularity as parameter name. You can find a guideline [here](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#singularity)

Reply:
> Please update the changelog base on this [PR](https://github.com/Azure/azure-powershell/pull/19608) @yadongyaly

Should I add change log on the "Comments" section of that PR?  I just added a comment for change log on your PR link

Reply:
@wyunchi-ms wyunchi-ms